ingest/validation/validate_metadata.py

```diff
@@ -1345,10 +1423,20 @@ def detect_excel_drag(metadata, convention):
                         )
                         if multiply_assigned:
                             msg += f"Check for mismatches between ontology ID and provided ontology label(s) {multiply_assigned}\n"
-                    metadata.store_validation_issue("error", "ontology", msg)
+                    metadata.store_validation_issue(
+                        "error",
+                        "ontology",
+                        msg,
+                        issue_name="content:metadata:mismatched-id-label",
```

Comment:
```suggestion
                        msg,
                        "ontology:id-label-mismatch",
```

Note `mismatch` at end, for consistency with other error types that contain the word "mismatch".

Reply:
Retiring `content:metadata:mismatched-id-label` for clarity, replacing with `ontology:multiply-assigned-label` which more clearly describes the "excel drag" indicator that was detected. This error is distinct from the ingest error `ontology:label-not-match-id`.